### PR TITLE
Whiteboard messages documentation reworked

### PR DESCRIPTION
**Pull request by comradekingu**
@orangesunny Looks good to me now.

### CODE REVIEW

File: docs/admin/whiteboard.rst [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
+This is how it looks on the language overview page:
 
 .. image:: /images/whiteboard-language.png
-
+   alt : Image showing a message that reads: "Czech translators rock!" atop the Czech language overview.
 
 And on the project page:
 
 .. image:: /images/whiteboard-project.png
+   alt : Image showing a message that reads: "Translations will be used only if they reach 60%" atop the dashboard view.
```
Comment (comradekingu) [inline docs/admin/whiteboard.rst]: This should be discouraged. Can't think of something to replace it right now. (Needs the image too).

File: docs/admin/whiteboard.rst [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
-    Shown on language overview and all translations in this language.
+    Shown on the language overview and all translations in that language.
 
 
-You can see how it looks on the language overview page:
+This is how it looks on the language overview page:
 
 .. image:: /images/whiteboard-language.png
-
+   alt : Image showing a message that reads: "Czech translators rock!" atop the Czech language overview.
```
Comment (nijel) [inline docs/admin/whiteboard.rst]:
```suggestion
   :alt: Image showing a message that reads: "Czech translators rock!" atop the Czech language overview.
```

File: docs/admin/whiteboard.rst [hunk 3]
```diff
@@ -1,40 +1,39 @@
...
+This is how it looks on the language overview page:
 
 .. image:: /images/whiteboard-language.png
-
+   :alt: Image showing a message that reads: "Czech translators rock!" atop the Czech language overview.
 
 And on the project page:
 
 .. image:: /images/whiteboard-project.png
+   alt : Image showing a message that reads: "Translations will be used only if they reach 60%" atop the dashboard view.
```
Comment (nijel) [inline docs/admin/whiteboard.rst]:
```suggestion
    :alt: Image showing a message that reads: "Translations will be used only if they reach 60%" atop the dashboard view.
```

File: docs/admin/whiteboard.rst [hunk 4]
```diff
@@ -1,40 +1,39 @@
 Whiteboard messages
 ===================
 
-You can use whiteboard messages to give some information to your translators.
-The message can be site-wide or targeted to a translation component or language.
+Provide infor to your translators.by posting messages, site-wide, per component or language.
```
Comment (comradekingu) [inline docs/admin/whiteboard.rst]:
```suggestion
Provide info to your translators.by posting messages, site-wide, per component or language.
```

Comment (comradekingu) [inline docs/admin/whiteboard.rst]:
```suggestion
Provide info to your translators.by posting messages, site-wide, per component, or language.
```

File: docs/admin/whiteboard.rst [hunk 5]
```diff
@@ -1,40 +1,39 @@
 Whiteboard messages
 ===================
 
-You can use whiteboard messages to give some information to your translators.
-The message can be site-wide or targeted to a translation component or language.
+Provide info to your translators.by posting messages, site-wide, per component, or language.
 
-This can be useful for various things from announcing the purpose of the website to
-specifying targets for translations.
+Announce the purpose, deadlines, status, or specify targets for translations.
```
Comment (comradekingu) [inline docs/admin/whiteboard.rst]:
```suggestion
Announce the purpose, deadlines, status, or specify targets for translation.
```

File: docs/admin/whiteboard.rst [hunk 6]
```diff
@@ -1,40 +1,39 @@
...
-You can use whiteboard messages to give some information to your translators.
-The message can be site-wide or targeted to a translation component or language.
+Provide info to your translators.by posting messages, site-wide, per component, or language.
 
-This can be useful for various things from announcing the purpose of the website to
-specifying targets for translations.
+Announce the purpose, deadlines, status, or specify targets for translation.
 
-The whiteboard can currently be specified only in the admin interface:
+Adding whiteboard messages can be done from the admin interface, or each respective cateogry for admins:
```
Comment (comradekingu) [inline docs/admin/whiteboard.rst]:
```suggestion
Adding whiteboard messages can be done from the admin interface, or in each respective category for admins:
```

File: docs/admin/whiteboard.rst [hunk 7]
```diff
@@ -1,40 +1,39 @@
...
+The whiteboard messages are then shown based on their specified context:
 
 No context specified
 
     Shown on dashboard (landing page).
 
 Project specified
 
-    Shown on project, all its components and translations.
+    Shown for project, and all its components and translations.
```
Comment (comradekingu) [inline docs/admin/whiteboard.rst]:
```suggestion
    Shown for project, including all its components and translations.
```

File: docs/admin/whiteboard.rst [hunk 8]
```diff
@@ -1,40 +1,39 @@
...
 
 Project specified
 
-    Shown on project, all its components and translations.
+    Shown for project, including all its components and translations.
 
 Component specified
 
-    Shown on component and all its translations.
+    Shown for a component and all its translations.
```
Comment (comradekingu) [inline docs/admin/whiteboard.rst]:
```suggestion
    Shown for a given component and all its translations.
```

File: docs/admin/whiteboard.rst [hunk 9]
```diff
@@ -1,40 +1,39 @@
...
+The whiteboard messages are then shown based on their specified context:
 
 No context specified
 
     Shown on dashboard (landing page).
 
 Project specified
 
-    Shown on project, all its components and translations.
+    Shown for project, including all its components and translations.
```
Comment (comradekingu) [inline docs/admin/whiteboard.rst:20]:
```suggestion
    Shown within the project, including all its components and translations.
```

**Comment by comradekingu** [comment]
@orangesunny I added a "shown within the project" as "shown for project" was a bit close-fitting.

**Comment by nijel** [comment]
I've merged this manually. As this got meanwhile renamed to announcements and docs was changed, I hope I didn't mess it up too much. See https://docs.weblate.org/en/latest/admin/announcements.html

**Pull request merged**